app/src/controllers/launch/selectors.js

```diff
@@ -24,3 +31,15 @@ export const queryParametersSelector = createQueryParametersSelector({
 });
 
 export const debugModeSelector = (state) => domainSelector(state).debugMode || false;
+
+const createLaunchesLinkSelector = (filterId) =>
+  createSelector(projectIdSelector, payloadSelector, (activeProject, payload) => ({
```

Comment:
Are you really need `projectIdSelector` here?

Reply:
ohh, really! It's already in the payload